src/styles/Select.ts

```diff
@@ -0,0 +1,28 @@
+import { Colors } from './Colors';
+
+export const SelectStyles = {
+  light: {
+    '&:focus': {
+      backgroundColor: Colors.SecondaryDefault,
+      paddingTop: '0px',
```

Comment:
Almost done! You only have to remove the` paddingTop and paddingBottom` light and dark styles for 'focus' and replace them with `minWidth: '160px'` This will eliminate the flashing on hover
<img width="1003" alt="Screenshot 2024-12-18 at 10 53 20" src="https://github.com/user-attachments/assets/8e507a05-e63f-4876-83bc-28ec0003ba58" />

Reply:
I still experience the flashing when I add `minWidth:'160px' ` to `focus`

https://github.com/user-attachments/assets/fa454379-32be-4d39-bc88-4bd68eaf7ea9

The only time I don't see the flashing is when I remove `paddingTop` and `paddingBottom` from both `focus` and `hover`(Hover and focus have only backgroundColor)

https://github.com/user-attachments/assets/668361a7-72a3-4da3-8d5b-0dbf382a1ae8

Reply:
That's fine for now. The different color on the padded left is for `on hover when focused` goes beyond the scope of this PR, which is to `keep it one solid color when focused`. We need to fix the padding of MuiInputBase to fix that clash and that can be a quick separate PR. 

Also, we have to leave the hover padding styles as is for now because of the issue I mentioned above:

![image](https://github.com/user-attachments/assets/2b8a1e70-46c6-4a20-bca9-f941260bee50)

Reply:
So once you push those final changes, we can get this merged. :)